clientlibs/js/package.json

```diff
@@ -7,7 +7,11 @@
   ],
   "scripts": {
     "build:bundler": "webpack",
-    "build:types": "./node_modules/.bin/dts-bundle-generator -o dist/browser/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/node/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/browser-lite/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/node-lite/index.d.ts src/index.ts",
+    "build:types": "concurrently \"npm run build:types-browser\" \"npm run build:types-node\" \"npm run build:types-browser-lite\" \"npm run build:types-node-lite\"",
```

Comment:
If you want to further optimise you can use this
`concurrently \"npm:build:types-*\"`

Reply:
implemented, works great